openedx/core/djangoapps/content_tagging/tests/test_api.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
             name="Learning Objectives",
             enabled=False,
         )
-        api.set_taxonomy_orgs(self.taxonomy_disabled, all_orgs=True)
+        api.set_taxonomy_orgs(self.taxonomy_disabled, orgs=[self.org1, self.org2])
```

Comment:
Why did this have to change? I think it's still useful to include a taxonomy in the tests which is disabled + all_orgs.

Reply:
I changed it to fix the following test:

https://github.com/open-craft/edx-platform/blob/aeeebf948941ed37c9cc8b28e15315e1d8d4f62a/openedx/core/djangoapps/content_tagging/tests/test_rules.py#L259C1-L278C57

We have disabled + all_orgs test here:
https://github.com/open-craft/edx-platform/blob/aeeebf948941ed37c9cc8b28e15315e1d8d4f62a/openedx/core/djangoapps/content_tagging/tests/test_rules.py#L375C1-L392C57